core/src/main/scala/com/spotify/featran/transformers/Binarizer.scala

```diff
@@ -52,4 +52,6 @@ private[featran] class Binarizer(name: String, val threshold: Double) extends Ma
     Map("threshold" -> threshold.toString)
 
   def flatRead[T: FlatReader]: T => Option[Any] = FlatReader[T].readDouble(name)
+  def flatWriter[T](implicit fw: FlatWriter[T]): Option[Double] => fw.IF =
```

Comment:
@richwhitjr should we have it `flatWriter[T: FlatWriter]`?

Reply:
This signature is weird.  If you notice I am using an internally declared type as the output the returned function.  Because of that I need access to the class in the signature to write the function.

Honestly surprised this worked.